src/main/java/org/coffeemine/app/spring/data/User.java

```diff
@@ -113,4 +113,8 @@ public User fromNO2Doc(Document doc) {
         account_passhash = doc.get("account_passhash", String.class);
         return this;
     }
+
+    public String toString(){
+        return name;
```

Comment:
this is just .getName();

Reply:
+1

Reply:
No really. Cause the multi-select box needs toString() to show the object name.

Reply:
Then use an inline proxy class